pkgs/development/python-modules/mmcv/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python3
+, pythonOlder
+, requests
+, cudaSupport ? false
+, cudatoolkit
+, torch-cuda-arch-list ? "Kepler"
+, torch
+, torchvision
+, torch-bin
+, torchvision-bin
+, opencv4
+, yapf
+, packaging
+, pillow
+, addict
+, ninja
+, which
+}:
+
+buildPythonPackage rec {
+  pname = "mmcv";
+  version = "1.7.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "open-mmlab";
+    repo = pname;
+    rev = "v1.7.0";
+    hash = "sha256-EVu6D6rTeebTKFCMNIbgQpvBS52TKk3vy2ReReJ9VQE=";
+  };
+
+  torch-deps = if cudaSupport then [ torch-bin torchvision-bin ] else [ torch torchvision ];
+  nativeBuildInputs = [ ninja which ]
+    ++ lib.lists.optionals cudaSupport [ cudatoolkit.lib cudatoolkit.out cudatoolkit.cc ] ++ torch-deps;
+
+  propagatedBuildInputs = [
+    opencv4
+    yapf
+    packaging
+    pillow
+    addict
+  ] ++ torch-deps;
+
+  doCheck = false;
```

Comment:
Please add a comment while the tests are not executed.

Reply:
have some of the tests running in the `checkPhase`.